packages/openapi-specification/openapi.yaml

```diff
@@ -1,742 +1,177 @@
-              $ref: '#/components/schemas/User'
-      description: List of user object
-      required: true
-    Pet:
-      content:
-        application/json:
-          schema:
-            $ref: '#/components/schemas/Pet'
-        application/xml:
-          schema:
-            $ref: '#/components/schemas/Pet'
-      description: Pet object that needs to be added to the store
-      required: true
   securitySchemes:
-    petstore_auth:
+    records_auth:
       type: oauth2
       flows:
         implicit:
-          authorizationUrl: 'http://petstore.swagger.io/api/oauth/dialog'
+          authorizationUrl: ''
           scopes:
-            'write:pets': modify pets in your account
-            'read:pets': read your pets
-    api_key:
-      type: apiKey
-      name: api_key
-      in: header
+            write:records: add and modify records in the website records list
+            read:records: read records from the website records list
+
+  requestBodies:
+    Record:
+      description: A record object
+      required: true
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Record'
+
   schemas:
-    Order:
-      title: Pet Order
-      description: An order for a pets from the pet store
-      type: object
-      properties:
-        id:
-          type: integer
-          format: int64
-        petId:
-          type: integer
-          format: int64
-        quantity:
-          type: integer
-          format: int32
-        shipDate:
-          type: string
-          format: date-time
-        status:
-          type: string
-          description: Order Status
-          enum:
-            - placed
-            - approved
-            - delivered
-        complete:
-          type: boolean
-          default: false
-      xml:
-        name: Order
-    Category:
-      title: Pet category
-      description: A category for a pet
-      type: object
-      properties:
-        id:
-          type: integer
-          format: int64
-        name:
-          type: string
-          pattern: '^[a-zA-Z0-9]+[a-zA-Z0-9\.\-_]*[a-zA-Z0-9]+$'
-      xml:
-        name: Category
-    User:
-      title: a User
-      description: A User who is purchasing from the pet store
+    Record:
+      title: Website Record
+      description: A record of a website crawl
       type: object
+      required:
+        - url
+        - boundaryRegEx
       properties:
         id:
           type: integer
           format: int64
-        username:
-          type: string
-        firstName:
-          type: string
-        lastName:
-          type: string
-        email:
-          type: string
-        password:
+        url:
+          description: The URL at which the crawler should start.
           type: string
-        phone:
+          format: uri
+          pattern: '^https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&\/=]*)$'
```

Comment:
the pattern is failing for simple websides like `hello.cz` and `https://hello.cz`. Do we need the pattern here?

Reply:
I fixed it so it doesn't require https:// up front, but the second one passed when I was testing it. We might not need the pattern, I'm just used to putting a pattern for urls